mozilla/blurts-server
# Add support for nimbus_web_preview HTTP parameter

Add support for nimbus_web_preview HTTP parameter proposed in https://github.com/mozilla/experimenter/pull/10902



# References:

Jira: MNTOR-3338



# Description

This causes Monitor to accept a `nimbus_web_preview` HTTP parameter on all pages that use `getExperiments()`, and makes `previewMode` boolean a required argument to `getExperiments`.

Currently this simply enables all features, but once https://github.com/mozilla/experimenter/pull/10902 is fully implemented then we'll pass that parameter to Cirrus which will cause it to use the Nimbus preview collection.

# How to test

This is a bit hard to test locally because `getExperiments` returns early when running locally, I used the debugger to make sure that the parameter was being set but it's probably easiest to test on stage.

# Checklist (Definition of Done)
